### PR TITLE
Add Corda essentials

### PR DESCRIPTION
**Pull request by akegaviar**
Add Corda essentials to Blockchains

### CODE REVIEW

File: docs/blockchains/corda.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+#### Network map service
+
+The network map service is the catalog of all nodes on the Corda network with the information on the node identities, node certificates, and node IP addresses.
+
+Each node on the Corda network keeps a copy of the network map.
+
+Each node entry in the network map is signed by the node it represents; this makes each entry tamper-proof.
+
+#### Identity service
```
Comment (easeev) [inline docs/blockchains/corda.md]: This seems to be related to [Corda Network](https://corda.network). Initially we are going to provide a [new dynamic availability zone](https://docs.corda.net/setting-up-a-dynamic-compatibility-zone.html) per network; support of Corda Testnet, Corda Network UAT or Corda Network is planned for the future.

Comment (akegaviar) [reply docs/blockchains/corda.md]: Fixed.

File: docs/blockchains/corda.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+* Node to node communication implemented as [AMQP 1.0](http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-overview-v1.0-os.html) over TLS using [Apache ActiveMQ Artemis](https://activemq.apache.org/components/artemis/).
+* Node to owner communication implemented as RPC.
+
+Each node keeps historic (consumed) and current (unconsumed) states of on-ledger facts shared between nodes participating in transactions. No node on the network has read access to the entirety of the ledger. Two or more nodes participating in a transaction always have the same read access to the subset of the ledger relevant to the transaction.
+
+Each node can be configured to be:
+
+* A regular node — a node with a certificate, two communication interfaces, and a ledger.
+* A node with CorDapp — a distributed application running a [contract](#contracts) and a [flow](/blockchains/corda#flows).
```
Comment (easeev) [inline docs/blockchains/corda.md]: You can install CorDapp on any regular node so it's not specific kind of node.

File: docs/blockchains/corda.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+A flow is an automated set of actions that a node runs on receiving an RPC request from the node owner.
+
+A flow is basically an automated business process split into a sequence of specific actions that the node owner does not have to manually initiate each time.
+
+#### Compatibility zones
+
+A compatibility zone is a deployed network instance. 
+
+There are three major types of compatibility zones:
```
Comment (easeev) [inline docs/blockchains/corda.md]: These are not types of compatibility zones, these are just different compatibility zones. So we might provide them just as an example.

File: docs/blockchains/corda.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+* [Corda Network](https://corda.network/) — Corda's main compatibility zone.
+* [Corda Testnet](https://testnet.corda.network/) — Corda's compatibility zone for testing.
+* Dynamic compatibility zones — a private compatibility zone. This is basically a privately deployed network that relies on a custom [network map server](#network-map-service) to allow nodes to join.
+
+Chainstack deploys every Corda network in dynamic compatibility zone. Every dynamic compatibility zone deployed with Chainstack comes with a network map server.
```
Comment (easeev) [inline docs/blockchains/corda.md]: Perhaps we should add a sentence that there will be an option to join a publicly available compatibility zone like Corda Network in the future.

**Pull request merged**